DocNet.Console/Program.cs

```diff
@@ -1,14 +1,146 @@
 
+        //Command Line Options
+        public class Options
+        {
+            [Option('i', "input", Required = true, HelpText = "File/Folder to convert to documentation.")]
+            public string InputFile { get; set; }
+            [Option('o', "output", Required = true, HelpText = "Location to output documentation.")]
+            public string OutputFile { get; set; }
+            [Option('r', "recurse", HelpText = "Recursively search through input folder", DefaultValue = false)]
+            public bool RecurseOption { get; set; }
+            [Option('h', "help", HelpText = "Display command line help information", Required = false, DefaultValue = false)]
+            public bool Help { get; set; }
+        }
+
+        //Checks that command line input is correct and outputs a custom message depending on whats missing
+        static private bool CmdCheck(string[] args)
+        {
+            //Determine which required commands are missing
+
+            if (args.Length == 0)
+            {
+                System.Console.WriteLine("No commands found. Use the following commands:");
+                return false;
+            }
+            //Check if args contains --help
+            else if ((Array.IndexOf(args, "-h") >= 0) || (Array.IndexOf(args, "--help") >= 0))
+            {
+                return false;
+            }
+            //Check if other commands exist
+            else if (!(((Array.IndexOf(args, "-o") >= 0) || (Array.IndexOf(args, "--output") >= 0)) && ((Array.IndexOf(args, "-i") >= 0) || (Array.IndexOf(args, "--input") >= 0))))
+            {
+                System.Console.WriteLine("Required commands not found. Use the following commands:");
+                return false;
+            }
+            /*else if(!((Array.IndexOf(args, "-i") >= 0) || (Array.IndexOf(args, "--input") >= 0)))
+            {
+                System.Console.WriteLine("Required output command not found. Use the following commands:");
+                return false;
+            }
+            else if (!((Array.IndexOf(args, "-o") >= 0) || (Array.IndexOf(args, "--output") >= 0)))
+            {
+               System.Console.WriteLine("Requried Input command not found. Use the following commands:");
+               return false;
+            } */
+            else
+            {
+                return true;
+            }
+        }
+        //Parses command line
+        static public void CommandParse(string[] args)
+        {
+            var options = new Options();
+            System.Console.WriteLine(options.OutputFile);
+            //Parse CL Input
+            if (CommandLine.Parser.Default.ParseArguments(args, options))
+            {
+                //Display help and return
+                if (options.Help)
+                {
+                    System.Console.WriteLine(CommandLine.Text.HelpText.AutoBuild(options));
+                    System.Console.ReadLine();
```

Comment:
You should make a TODO comment here to remove the call to ReadLine(). Tip: ReSharper will highlight comments beginning with "TODO" in blue to make them more visible.